LoRaEngine/test/LoRaWan.IntegrationTest/ABPTest.cs

```diff
@@ -220,6 +220,79 @@ public async Task Test_ABP_Invalid_NwkSKey_Fails_With_Mic_Error()
 
             // Before starting new test, wait until Lora drivers stops sending/receiving data
             await this.ArduinoDevice.WaitForIdleAsync();
-        }        
+        }
+
+        // Verifies that ABP confirmed and unconfirmed messages are working
+        // Uses Device5_ABP
+        [Fact]
+        public async Task Test_ABP_Device_With_Same_DevAddr()
+        {
+            const int MESSAGES_COUNT = 2;
+
+            await SendABPMessages(MESSAGES_COUNT, this.TestFixture.Device16_ABP);
+            await SendABPMessages(MESSAGES_COUNT, this.TestFixture.Device17_ABP);
+
+        }
+
+        private async Task SendABPMessages(int MESSAGES_COUNT, TestDeviceInfo device)
+        {
+            Log($"[INFO] ** Starting {nameof(Test_ABP_Confirmed_And_Unconfirmed_Message)} using device {device.DeviceID} **");
```

Comment:
This is the wrong place. Should be where test (with Fact attribute) is.
Log($"[INFO] ** Starting {nameof(Test_ABP_Device_With_Same_DevAddr)} using devices {this.TestFixture.Device16_ABP.DeviceID}, {this.TestFixture.Device17_ABP.DeviceID} **");